Content.Shared/Clothing/Components/ClothingComponent.cs

```diff
@@ -66,6 +66,9 @@ public sealed partial class ClothingComponent : Component
     [DataField("unisexMask")]
     public ClothingMask UnisexMask = ClothingMask.UniformFull;
 
+    [DataField("isEquipped")]
+    public bool IsEquipped;
+
```

Comment:
This should not be here. Check `InSlot != null` instead. Introducing tautologies like this will lead to new edge cases in the future that this PR cannot account for.

Reply:
Gotcha, couldnt find something like that when I looked but its good to know its there!